themes/default/content/challenge/holiday-shopping.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Holiday Shopping"
+layout: challenge/single
+description: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas 
+meta_desc: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas
+meta_image: /images/challenge/challenge_cta.png
+---
+
+## Welcome to the Pulumi Challenge!
+
+<div class="flex flex-wrap md:mt-12">
```

Comment:
We don't need all this HTML in here -- we can just use straight Markdown. (Earlier versions of this had imagery here, I think, which is probably where this comes from.)